### PR TITLE
Esther

### PR DESCRIPTION
**Pull request by washucode**
I updated the Border Radius. Border Radii was not working .

### CODE REVIEW

File: figma-tokens.json [hunk 1]
```diff
@@ -2,6 +2,10 @@
   "core": {
     "colors": {
       "neutral": {
+        "0": {
+          "value": "transparent",
+          "type": "color"
```
Comment (Ugizwenayo-Divine) [inline figma-tokens.json:7]: Well done @washucode  would you consider renaming the type to colors please if possible

File: figma-tokens.json [hunk 2]
```diff
@@ -345,6 +353,40 @@
         "value": "9999",
         "type": "borderRadii"
       }
+    },
+    "borderRadius": {
```
Comment (Ugizwenayo-Divine) [inline figma-tokens.json]: I'm wondering if it is possible to rename this borderRadius to borderRadii, we should not have both borderReadius and borderRadii

File: figma-tokens.json [hunk 3]
```diff
@@ -480,6 +522,84 @@
           "value": "{borderRadii.3}",
           "type": "borderRadii"
         }
+      },
+      "sm": {
+        "value": "{borderRadius.md}",
+        "type": "borderRadius"
```
Comment (Ugizwenayo-Divine) [inline figma-tokens.json]: Is it possible to rename this borderRadius type to borderRadii? if not we will need to refactor

File: figma-tokens.json [hunk 4]
```diff
@@ -480,6 +522,84 @@
...
+      "xl": {
+        "value": "{borderRadius.4}",
+        "type": "borderRadius"
+      },
+      "2xl": {
+        "value": "{borderRadius.6}",
+        "type": "borderRadius"
+      }
+    },
+    "space": {
```
Comment (Ugizwenayo-Divine) [inline figma-tokens.json]: Also rename this space to spacing

**Comment by Jaman-dedy** [comment]
Good job @washucode, please consider having this convention for your naming the branch ex : `ft-add-avatar-component`

File: figma-tokens.json [hunk 5]
```diff
@@ -614,5 +582,127 @@
...
+      },
+      "lg": {
+        "value": "{spacing.5}",
+        "type": "spacing"
+      }
+    },
+    "background": {
+      "solid": {
+        "value": "{colors.bg.solid}",
+        "type": "color"
```
Comment (Ugizwenayo-Divine) [inline figma-tokens.json]: I believe the type here should be colors

File: figma-tokens.json [hunk 6]
```diff
@@ -614,5 +582,127 @@
...
+      }
+    },
+    "borderRadius": {
+      "md": {
+        "value": "{borderRadii.2}",
+        "type": "borderRadius"
+      },
+      "lg": {
+        "value": "{borderRadii.3}",
+        "type": "borderRadius"
```
Comment (Ugizwenayo-Divine) [inline figma-tokens.json]: here the type should be borderRadii

File: figma-tokens.json [hunk 7]
```diff
@@ -407,80 +445,6 @@
           "type": "colors"
         }
       }
-    },
-    "size": {
```
Comment (Ugizwenayo-Divine) [inline figma-tokens.json:411]: These tokens are deleted but still being referenced in the avatar, icon and currency icon component, please cross check on it

File: figma-tokens.json [hunk 8]
```diff
@@ -599,20 +567,142 @@
...
-          "type": "borderRadii"
+          "value": "{borderRadii.3}",
+          "type": "borderRadius"
         }
       }
     }
-  }
-}
+  },
+  "component.button": {
```
Comment (Ugizwenayo-Divine) [inline figma-tokens.json:582]: would you please follow the component structure where we wrap all tokens in component name
Screen Shot 2022-08-03 at 09 12 33

**Review by Ugizwenayo-Divine — COMMENTED** [review]
in button component we are referencing padding.md which is not defined in core theme
Screen Shot 2022-08-03 at 09 17 30

File: figma-tokens.json [hunk 9]
```diff
@@ -2,6 +2,10 @@
   "core": {
     "colors": {
       "neutral": {
+        "0": {
+          "value": "transparent",
+          "type": "colors"
```
Comment (Ugizwenayo-Divine) [inline figma-tokens.json]: Well done @washucode 
is possible to change this type to color instead of colors

File: figma-tokens.json [hunk 10]
```diff
@@ -2,6 +2,10 @@
...
+        },
         "10": {
           "value": "#ffffff",
           "type": "colors"
```
Comment (Ugizwenayo-Divine) [inline figma-tokens.json]: same here change type to color instead of colors

File: figma-tokens.json [hunk 11]
```diff
@@ -509,38 +473,42 @@
         "onSolid": {
           "value": "{colors.neutral.120}",
           "type": "colors"
```
Comment (Ugizwenayo-Divine) [inline figma-tokens.json]: Same here change type to color instead of colors

File: figma-tokens.json [hunk 12]
```diff
@@ -345,140 +349,100 @@
         "value": "9999",
         "type": "borderRadii"
       }
+    },
+    "fontSize": {
+      "xs": {
+        "value": "12px",
```
Comment (Ugizwenayo-Divine) [inline figma-tokens.json]: We agreed to use numbers, would consider using just numbers in place of pixels

**Review by Ugizwenayo-Divine — COMMENTED** [review]
it looks good tho letter spacing is still using %, would you mind refactoring letterspacing in core

**Comment by Jaman-dedy** [comment]
LGTM

**Review by Jaman-dedy — APPROVED** [review]
LGTM

**Pull request merged**